### PR TITLE
Revert last changes and bump version to 0.0.30

### PR DESCRIPTION
**Pull request by rom10**

### CODE REVIEW

File: dazel.py [hunk 1]
```diff
@@ -17,7 +14,7 @@
 DEFAULT_IMAGE_NAME = "dazel"
 DEFAULT_RUN_COMMAND = "/bin/bash"
 DEFAULT_LOCAL_DOCKERFILE = "Dockerfile.dazel"
-DEFAULT_REMOTE_REPOSITORY = "dazel"
+DEFAULT_REMOTE_RPOSITORY = "dazel"
```
Comment (mishas) [inline dazel.py]: Mistake?

File: dazel.py [hunk 2]
```diff
@@ -100,7 +94,7 @@ def from_config(cls):
                 image_name=config.get("DAZEL_IMAGE_NAME", DEFAULT_IMAGE_NAME),
                 run_command=config.get("DAZEL_RUN_COMMAND", DEFAULT_RUN_COMMAND),
                 dockerfile=config.get("DAZEL_DOCKERFILE", DEFAULT_LOCAL_DOCKERFILE),
-                repository=config.get("DAZEL_REPOSITORY", DEFAULT_REMOTE_REPOSITORY),
+                repository=config.get("DAZEL_REPOSITORY", DEFAULT_REMOTE_RPOSITORY),
```
Comment (mishas) [inline dazel.py]: Here too

File: dazel.py (hunk 1 above)
Comment (rom10) [inline dazel.py]: Done

File: dazel.py (hunk 2 above)
Comment (rom10) [inline dazel.py]: Done

**Pull request merged**